akka-docs-new/src/main/paradox/guide/introduction.md

```diff
@@ -0,0 +1,49 @@
+# Introduction to Akka
+
+Welcome to Akka, a set of open-source libraries for designing scalable, resilient systems that 
+span processor cores and networks. Akka allows you to focus on meeting business needs instead 
+of writing low-level code to provide reliable behavior, fault tolerance, and high performance.
+
+Common practices and programming models do not address important challenges inherent in designing systems 
+for modern computer architectures. To be successful, distributed systems must cope in an environment where components 
+crash without responding, messages get lost without a trace on the wire, and network latency fluctuates. 
+These problems occur regularly in carefully managed intra-datacenter environments - even more so in virtualized 
+architectures.
+
+To deal with these realities, Akka provides
```

Comment:
No, actually the bullet points should start with small letters and end with commas except the last one (it is still regarded as a sentence).